_posts/2016-02-10-solita-developer-manifesto.md

```diff
@@ -0,0 +1,41 @@
+
+#### Learning new things and techniques over sticking to what you know
+#### Professional growth over staying in your comfort zone
+Probably the thing I love about Solita the most, is the fact that we’re constantly encouraged to share information and help our colleagues learn. One of my criteria when I was looking for a job some time ago, was that I wanted to join a company where I could feel like I can learn from my coworkers, and this is a promise that Solita has fulfilled hands down. For example, each of our three offices organizes monthly intra-office meetings, where the topic is somehow related to functional programming (dubbed *FP Monthlies*). These are just a drop in the bucket, as sometimes it feels like I would literally have no time to work on my project if I was to accept every invite to every meeting.
+
+#### Delivering customer value over unnecessary technical gimmicks
+
+This is, perhaps, an easily misunderstood principle. As developers we often find joy in trying out cool new tools and libraries, or building elegant abstractions that can handle any otherworldly corner case we can think of. While it is important that we are willing to learn and are passionate about the work we do, we should always try to be working on things that the customer can be excited about also, or at least on something that will later help us deliver customer value. Note that this principle does not mean that we shouldn’t focus on things like improving our tooling, removing technical debt, or trying out new but promising technologies, as it should be clear that all of these can help us do what is important: deliver customer value.
+
+#### Professional software development over hacking for fun
+#### Enjoying professional software development over just hacking for fun
+#### Combining fun, talent & professionalism over just hacking for fun
+
+Another principle that has the potential to be misinterpreted by the developer community, this one saw the most rework out of the four after the Solita Developer Manifesto was unveiled at the company’s internal, day-long developer conference. After some admittedly beer-fueled conversations, the principle got its current form, and much like the previous principle, it now helps us define what we expect of our colleagues when working on a customer project. While programming is fun, and should be fun, we are all software development professionals, and that entails a lot more than just hacking away. Meetings, emails and phone calls, are all things that clearly fall out of the circle of “cool tech things”, but are nevertheless an integral part of any project. Continuous integration, testing and code review practices, or security and performance testing may be things that do not really interest you, but someone in your team has to take care of these things.
```

Comment:
its -> it's
software development professionals -> professional software developers ? hivenen erilainen vivahde merkitykseen tulee, joten on vähän makuasia kumpi on parempi.